2019-12-25 09:52:12.155  INFO 6882 --- [DubboZkclientConnector-SendThread(122.51.6.138:2181)] org.apache.zookeeper.ClientCnxn          : Opening socket connection to server 122.51.6.138/122.51.6.138:2181. Will not attempt to authenticate using SASL (unknown error)
2019-12-25 09:52:12.189 ERROR 6882 --- [DubboZkclientConnector-SendThread(122.51.6.138:2181)] o.apache.zookeeper.ClientCnxnSocketNIO   : Unable to open socket to 122.51.6.138/122.51.6.138:2181
2019-12-25 09:52:12.195  WARN 6882 --- [DubboZkclientConnector-SendThread(122.51.6.138:2181)] org.apache.zookeeper.ClientCnxn          : Session 0x10000004c5e0058 for server null, unexpected error, closing socket connection and attempting reconnect

java.net.SocketException: Network is unreachable
	at java.base/sun.nio.ch.Net.connect0(Native Method) ~[na:na]
	at java.base/sun.nio.ch.Net.connect(Net.java:482) ~[na:na]
	at java.base/sun.nio.ch.Net.connect(Net.java:474) ~[na:na]
	at java.base/sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:694) ~[na:na]
	at org.apache.zookeeper.ClientCnxnSocketNIO.registerAndConnect(ClientCnxnSocketNIO.java:277) ~[zookeeper-3.4.8.jar:3.4.8--1]
	at org.apache.zookeeper.ClientCnxnSocketNIO.connect(ClientCnxnSocketNIO.java:287) ~[zookeeper-3.4.8.jar:3.4.8--1]
	at org.apache.zookeeper.ClientCnxn$SendThread.startConnect(ClientCnxn.java:1024) ~[zookeeper-3.4.8.jar:3.4.8--1]
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1060) ~[zookeeper-3.4.8.jar:3.4.8--1]

2019-12-25 09:52:14.153  INFO 6882 --- [DubboZkclientConnector-SendThread(122.51.6.138:2181)] org.apache.zookeeper.ClientCnxn          : Opening socket connection to server 122.51.6.138/122.51.6.138:2181. Will not attempt to authenticate using SASL (unknown error)
2019-12-25 09:52:14.154 ERROR 6882 --- [DubboZkclientConnector-SendThread(122.51.6.138:2181)] o.apache.zookeeper.ClientCnxnSocketNIO   : Unable to open socket to 122.51.6.138/122.51.6.138:2181
2019-12-25 09:52:14.154  WARN 6882 --- [DubboZkclientConnector-SendThread(122.51.6.138:2181)] org.apache.zookeeper.ClientCnxn          : Session 0x10000004c5e0058 for server null, unexpected error, closing socket connection and attempting reconnect

java.net.SocketException: Network is unreachable
	at java.base/sun.nio.ch.Net.connect0(Native Method) ~[na:na]
	at java.base/sun.nio.ch.Net.connect(Net.java:482) ~[na:na]
	at java.base/sun.nio.ch.Net.connect(Net.java:474) ~[na:na]
	at java.base/sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:694) ~[na:na]
	at org.apache.zookeeper.ClientCnxnSocketNIO.registerAndConnect(ClientCnxnSocketNIO.java:277) ~[zookeeper-3.4.8.jar:3.4.8--1]
	at org.apache.zookeeper.ClientCnxnSocketNIO.connect(ClientCnxnSocketNIO.java:287) ~[zookeeper-3.4.8.jar:3.4.8--1]
	at org.apache.zookeeper.ClientCnxn$SendThread.startConnect(ClientCnxn.java:1024) ~[zookeeper-3.4.8.jar:3.4.8--1]
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1060) ~[zookeeper-3.4.8.jar:3.4.8--1]

2019-12-25 09:52:15.576  INFO 6882 --- [DubboZkclientConnector-SendThread(122.51.6.138:2181)] org.apache.zookeeper.ClientCnxn          : Opening socket connection to server 122.51.6.138/122.51.6.138:2181. Will not attempt to authenticate using SASL (unknown error)
2019-12-25 09:52:15.577 ERROR 6882 --- [DubboZkclientConnector-SendThread(122.51.6.138:2181)] o.apache.zookeeper.ClientCnxnSocketNIO   : Unable to open socket to 122.51.6.138/122.51.6.138:2181
2019-12-25 09:52:15.577  WARN 6882 --- [DubboZkclientConnector-SendThread(122.51.6.138:2181)] org.apache.zookeeper.ClientCnxn          : Session 0x10000004c5e0058 for server null, unexpected error, closing socket connection and attempting reconnect

java.net.SocketException: Network is unreachable
	at java.base/sun.nio.ch.Net.connect0(Native Method) ~[na:na]
	at java.base/sun.nio.ch.Net.connect(Net.java:482) ~[na:na]
	at java.base/sun.nio.ch.Net.connect(Net.java:474) ~[na:na]
	at java.base/sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:694) ~[na:na]
	at org.apache.zookeeper.ClientCnxnSocketNIO.registerAndConnect(ClientCnxnSocketNIO.java:277) ~[zookeeper-3.4.8.jar:3.4.8--1]
	at org.apache.zookeeper.ClientCnxnSocketNIO.connect(ClientCnxnSocketNIO.java:287) ~[zookeeper-3.4.8.jar:3.4.8--1]
	at org.apache.zookeeper.ClientCnxn$SendThread.startConnect(ClientCnxn.java:1024) ~[zookeeper-3.4.8.jar:3.4.8--1]
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1060) ~[zookeeper-3.4.8.jar:3.4.8--1]

2019-12-25 09:52:16.818  INFO 6882 --- [DubboZkclientConnector-SendThread(122.51.6.138:2181)] org.apache.zookeeper.ClientCnxn          : Opening socket connection to server 122.51.6.138/122.51.6.138:2181. Will not attempt to authenticate using SASL (unknown error)
2019-12-25 09:52:16.818 ERROR 6882 --- [DubboZkclientConnector-SendThread(122.51.6.138:2181)] o.apache.zookeeper.ClientCnxnSocketNIO   : Unable to open socket to 122.51.6.138/122.51.6.138:2181
2019-12-25 09:52:16.819  WARN 6882 --- [DubboZkclientConnector-SendThread(122.51.6.138:2181)] org.apache.zookeeper.ClientCnxn          : Session 0x10000004c5e0058 for server null, unexpected error, closing socket connection and attempting reconnect

java.net.SocketException: Network is unreachable
	at java.base/sun.nio.ch.Net.connect0(Native Method) ~[na:na]
	at java.base/sun.nio.ch.Net.connect(Net.java:482) ~[na:na]
	at java.base/sun.nio.ch.Net.connect(Net.java:474) ~[na:na]
	at java.base/sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:694) ~[na:na]
	at org.apache.zookeeper.ClientCnxnSocketNIO.registerAndConnect(ClientCnxnSocketNIO.java:277) ~[zookeeper-3.4.8.jar:3.4.8--1]
	at org.apache.zookeeper.ClientCnxnSocketNIO.connect(ClientCnxnSocketNIO.java:287) ~[zookeeper-3.4.8.jar:3.4.8--1]
	at org.apache.zookeeper.ClientCnxn$SendThread.startConnect(ClientCnxn.java:1024) ~[zookeeper-3.4.8.jar:3.4.8--1]
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1060) ~[zookeeper-3.4.8.jar:3.4.8--1]

2019-12-25 09:52:18.833  INFO 6882 --- [DubboZkclientConnector-SendThread(122.51.6.138:2181)] org.apache.zookeeper.ClientCnxn          : Opening socket connection to server 122.51.6.138/122.51.6.138:2181. Will not attempt to authenticate using SASL (unknown error)
2019-12-25 09:52:18.834 ERROR 6882 --- [DubboZkclientConnector-SendThread(122.51.6.138:2181)] o.apache.zookeeper.ClientCnxnSocketNIO   : Unable to open socket to 122.51.6.138/122.51.6.138:2181
2019-12-25 09:52:18.834  WARN 6882 --- [DubboZkclientConnector-SendThread(122.51.6.138:2181)] org.apache.zookeeper.ClientCnxn          : Session 0x10000004c5e0058 for server null, unexpected error, closing socket connection and attempting reconnect

java.net.SocketException: Network is unreachable
	at java.base/sun.nio.ch.Net.connect0(Native Method) ~[na:na]
	at java.base/sun.nio.ch.Net.connect(Net.java:482) ~[na:na]
	at java.base/sun.nio.ch.Net.connect(Net.java:474) ~[na:na]
	at java.base/sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:694) ~[na:na]
	at org.apache.zookeeper.ClientCnxnSocketNIO.registerAndConnect(ClientCnxnSocketNIO.java:277) ~[zookeeper-3.4.8.jar:3.4.8--1]
	at org.apache.zookeeper.ClientCnxnSocketNIO.connect(ClientCnxnSocketNIO.java:287) ~[zookeeper-3.4.8.jar:3.4.8--1]
	at org.apache.zookeeper.ClientCnxn$SendThread.startConnect(ClientCnxn.java:1024) ~[zookeeper-3.4.8.jar:3.4.8--1]
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1060) ~[zookeeper-3.4.8.jar:3.4.8--1]

2019-12-25 09:52:20.798  INFO 6882 --- [DubboZkclientConnector-SendThread(122.51.6.138:2181)] org.apache.zookeeper.ClientCnxn          : Opening socket connection to server 122.51.6.138/122.51.6.138:2181. Will not attempt to authenticate using SASL (unknown error)
2019-12-25 09:52:20.799 ERROR 6882 --- [DubboZkclientConnector-SendThread(122.51.6.138:2181)] o.apache.zookeeper.ClientCnxnSocketNIO   : Unable to open socket to 122.51.6.138/122.51.6.138:2181
2019-12-25 09:52:20.799  WARN 6882 --- [DubboZkclientConnector-SendThread(122.51.6.138:2181)] org.apache.zookeeper.ClientCnxn          : Session 0x10000004c5e0058 for server null, unexpected error, closing socket connection and attempting reconnect

java.net.SocketException: Network is unreachable
	at java.base/sun.nio.ch.Net.connect0(Native Method) ~[na:na]
	at java.base/sun.nio.ch.Net.connect(Net.java:482) ~[na:na]
	at java.base/sun.nio.ch.Net.connect(Net.java:474) ~[na:na]
	at java.base/sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:694) ~[na:na]
	at org.apache.zookeeper.ClientCnxnSocketNIO.registerAndConnect(ClientCnxnSocketNIO.java:277) ~[zookeeper-3.4.8.jar:3.4.8--1]
	at org.apache.zookeeper.ClientCnxnSocketNIO.connect(ClientCnxnSocketNIO.java:287) ~[zookeeper-3.4.8.jar:3.4.8--1]
	at org.apache.zookeeper.ClientCnxn$SendThread.startConnect(ClientCnxn.java:1024) ~[zookeeper-3.4.8.jar:3.4.8--1]
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1060) ~[zookeeper-3.4.8.jar:3.4.8--1]

2019-12-25 09:52:22.429  INFO 6882 --- [DubboZkclientConnector-SendThread(122.51.6.138:2181)] org.apache.zookeeper.ClientCnxn          : Opening socket connection to server 122.51.6.138/122.51.6.138:2181. Will not attempt to authenticate using SASL (unknown error)
2019-12-25 09:52:22.429 ERROR 6882 --- [DubboZkclientConnector-SendThread(122.51.6.138:2181)] o.apache.zookeeper.ClientCnxnSocketNIO   : Unable to open socket to 122.51.6.138/122.51.6.138:2181
2019-12-25 09:52:22.429  WARN 6882 --- [DubboZkclientConnector-SendThread(122.51.6.138:2181)] org.apache.zookeeper.ClientCnxn          : Session 0x10000004c5e0058 for server null, unexpected error, closing socket connection and attempting reconnect

java.net.SocketException: Network is unreachable
	at java.base/sun.nio.ch.Net.connect0(Native Method) ~[na:na]
	at java.base/sun.nio.ch.Net.connect(Net.java:482) ~[na:na]
	at java.base/sun.nio.ch.Net.connect(Net.java:474) ~[na:na]
	at java.base/sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:694) ~[na:na]
	at org.apache.zookeeper.ClientCnxnSocketNIO.registerAndConnect(ClientCnxnSocketNIO.java:277) ~[zookeeper-3.4.8.jar:3.4.8--1]
	at org.apache.zookeeper.ClientCnxnSocketNIO.connect(ClientCnxnSocketNIO.java:287) ~[zookeeper-3.4.8.jar:3.4.8--1]
	at org.apache.zookeeper.ClientCnxn$SendThread.startConnect(ClientCnxn.java:1024) ~[zookeeper-3.4.8.jar:3.4.8--1]
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1060) ~[zookeeper-3.4.8.jar:3.4.8--1]

2019-12-25 09:52:23.782  INFO 6882 --- [DubboZkclientConnector-SendThread(122.51.6.138:2181)] org.apache.zookeeper.ClientCnxn          : Opening socket connection to server 122.51.6.138/122.51.6.138:2181. Will not attempt to authenticate using SASL (unknown error)
2019-12-25 09:52:23.782 ERROR 6882 --- [DubboZkclientConnector-SendThread(122.51.6.138:2181)] o.apache.zookeeper.ClientCnxnSocketNIO   : Unable to open socket to 122.51.6.138/122.51.6.138:2181
2019-12-25 09:52:23.784  WARN 6882 --- [DubboZkclientConnector-SendThread(122.51.6.138:2181)] org.apache.zookeeper.ClientCnxn          : Session 0x10000004c5e0058 for server null, unexpected error, closing socket connection and attempting reconnect

java.net.SocketException: Network is unreachable
	at java.base/sun.nio.ch.Net.connect0(Native Method) ~[na:na]
	at java.base/sun.nio.ch.Net.connect(Net.java:482) ~[na:na]
	at java.base/sun.nio.ch.Net.connect(Net.java:474) ~[na:na]
	at java.base/sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:694) ~[na:na]
	at org.apache.zookeeper.ClientCnxnSocketNIO.registerAndConnect(ClientCnxnSocketNIO.java:277) ~[zookeeper-3.4.8.jar:3.4.8--1]
	at org.apache.zookeeper.ClientCnxnSocketNIO.connect(ClientCnxnSocketNIO.java:287) ~[zookeeper-3.4.8.jar:3.4.8--1]
	at org.apache.zookeeper.ClientCnxn$SendThread.startConnect(ClientCnxn.java:1024) ~[zookeeper-3.4.8.jar:3.4.8--1]
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1060) ~[zookeeper-3.4.8.jar:3.4.8--1]

2019-12-25 09:52:25.064  INFO 6882 --- [DubboZkclientConnector-SendThread(122.51.6.138:2181)] org.apache.zookeeper.ClientCnxn          : Opening socket connection to server 122.51.6.138/122.51.6.138:2181. Will not attempt to authenticate using SASL (unknown error)
2019-12-25 09:52:25.064 ERROR 6882 --- [DubboZkclientConnector-SendThread(122.51.6.138:2181)] o.apache.zookeeper.ClientCnxnSocketNIO   : Unable to open socket to 122.51.6.138/122.51.6.138:2181
2019-12-25 09:52:25.065  WARN 6882 --- [DubboZkclientConnector-SendThread(122.51.6.138:2181)] org.apache.zookeeper.ClientCnxn          : Session 0x10000004c5e0058 for server null, unexpected error, closing socket connection and attempting reconnect

java.net.SocketException: Network is unreachable
	at java.base/sun.nio.ch.Net.connect0(Native Method) ~[na:na]
	at java.base/sun.nio.ch.Net.connect(Net.java:482) ~[na:na]
	at java.base/sun.nio.ch.Net.connect(Net.java:474) ~[na:na]
	at java.base/sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:694) ~[na:na]
	at org.apache.zookeeper.ClientCnxnSocketNIO.registerAndConnect(ClientCnxnSocketNIO.java:277) ~[zookeeper-3.4.8.jar:3.4.8--1]
	at org.apache.zookeeper.ClientCnxnSocketNIO.connect(ClientCnxnSocketNIO.java:287) ~[zookeeper-3.4.8.jar:3.4.8--1]
	at org.apache.zookeeper.ClientCnxn$SendThread.startConnect(ClientCnxn.java:1024) ~[zookeeper-3.4.8.jar:3.4.8--1]
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1060) ~[zookeeper-3.4.8.jar:3.4.8--1]

2019-12-25 09:52:26.743  INFO 6882 --- [DubboZkclientConnector-SendThread(122.51.6.138:2181)] org.apache.zookeeper.ClientCnxn          : Opening socket connection to server 122.51.6.138/122.51.6.138:2181. Will not attempt to authenticate using SASL (unknown error)
2019-12-25 09:52:26.744 ERROR 6882 --- [DubboZkclientConnector-SendThread(122.51.6.138:2181)] o.apache.zookeeper.ClientCnxnSocketNIO   : Unable to open socket to 122.51.6.138/122.51.6.138:2181
2019-12-25 09:52:26.744  WARN 6882 --- [DubboZkclientConnector-SendThread(122.51.6.138:2181)] org.apache.zookeeper.ClientCnxn          : Session 0x10000004c5e0058 for server null, unexpected error, closing socket connection and attempting reconnect

java.net.SocketException: Network is unreachable
	at java.base/sun.nio.ch.Net.connect0(Native Method) ~[na:na]
	at java.base/sun.nio.ch.Net.connect(Net.java:482) ~[na:na]
	at java.base/sun.nio.ch.Net.connect(Net.java:474) ~[na:na]
	at java.base/sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:694) ~[na:na]
	at org.apache.zookeeper.ClientCnxnSocketNIO.registerAndConnect(ClientCnxnSocketNIO.java:277) ~[zookeeper-3.4.8.jar:3.4.8--1]
	at org.apache.zookeeper.ClientCnxnSocketNIO.connect(ClientCnxnSocketNIO.java:287) ~[zookeeper-3.4.8.jar:3.4.8--1]
	at org.apache.zookeeper.ClientCnxn$SendThread.startConnect(ClientCnxn.java:1024) ~[zookeeper-3.4.8.jar:3.4.8--1]
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1060) ~[zookeeper-3.4.8.jar:3.4.8--1]

2019-12-25 09:52:28.484  INFO 6882 --- [DubboZkclientConnector-SendThread(122.51.6.138:2181)] org.apache.zookeeper.ClientCnxn          : Opening socket connection to server 122.51.6.138/122.51.6.138:2181. Will not attempt to authenticate using SASL (unknown error)
2019-12-25 09:52:28.485 ERROR 6882 --- [DubboZkclientConnector-SendThread(122.51.6.138:2181)] o.apache.zookeeper.ClientCnxnSocketNIO   : Unable to open socket to 122.51.6.138/122.51.6.138:2181
2019-12-25 09:52:28.485  WARN 6882 --- [DubboZkclientConnector-SendThread(122.51.6.138:2181)] org.apache.zookeeper.ClientCnxn          : Session 0x10000004c5e0058 for server null, unexpected error, closing socket connection and attempting reconnect

java.net.SocketException: Network is unreachable
	at java.base/sun.nio.ch.Net.connect0(Native Method) ~[na:na]
	at java.base/sun.nio.ch.Net.connect(Net.java:482) ~[na:na]
	at java.base/sun.nio.ch.Net.connect(Net.java:474) ~[na:na]
	at java.base/sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:694) ~[na:na]
	at org.apache.zookeeper.ClientCnxnSocketNIO.registerAndConnect(ClientCnxnSocketNIO.java:277) ~[zookeeper-3.4.8.jar:3.4.8--1]
	at org.apache.zookeeper.ClientCnxnSocketNIO.connect(ClientCnxnSocketNIO.java:287) ~[zookeeper-3.4.8.jar:3.4.8--1]
	at org.apache.zookeeper.ClientCnxn$SendThread.startConnect(ClientCnxn.java:1024) ~[zookeeper-3.4.8.jar:3.4.8--1]
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1060) ~[zookeeper-3.4.8.jar:3.4.8--1]

2019-12-25 09:52:30.493  INFO 6882 --- [DubboZkclientConnector-SendThread(122.51.6.138:2181)] org.apache.zookeeper.ClientCnxn          : Opening socket connection to server 122.51.6.138/122.51.6.138:2181. Will not attempt to authenticate using SASL (unknown error)
2019-12-25 09:52:30.494 ERROR 6882 --- [DubboZkclientConnector-SendThread(122.51.6.138:2181)] o.apache.zookeeper.ClientCnxnSocketNIO   : Unable to open socket to 122.51.6.138/122.51.6.138:2181
2019-12-25 09:52:30.494  WARN 6882 --- [DubboZkclientConnector-SendThread(122.51.6.138:2181)] org.apache.zookeeper.ClientCnxn          : Session 0x10000004c5e0058 for server null, unexpected error, closing socket connection and attempting reconnect

java.net.SocketException: Network is unreachable
	at java.base/sun.nio.ch.Net.connect0(Native Method) ~[na:na]
	at java.base/sun.nio.ch.Net.connect(Net.java:482) ~[na:na]
	at java.base/sun.nio.ch.Net.connect(Net.java:474) ~[na:na]
	at java.base/sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:694) ~[na:na]
	at org.apache.zookeeper.ClientCnxnSocketNIO.registerAndConnect(ClientCnxnSocketNIO.java:277) ~[zookeeper-3.4.8.jar:3.4.8--1]
	at org.apache.zookeeper.ClientCnxnSocketNIO.connect(ClientCnxnSocketNIO.java:287) ~[zookeeper-3.4.8.jar:3.4.8--1]
	at org.apache.zookeeper.ClientCnxn$SendThread.startConnect(ClientCnxn.java:1024) ~[zookeeper-3.4.8.jar:3.4.8--1]
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1060) ~[zookeeper-3.4.8.jar:3.4.8--1]

2019-12-25 09:52:31.629  INFO 6882 --- [DubboZkclientConnector-SendThread(122.51.6.138:2181)] org.apache.zookeeper.ClientCnxn          : Opening socket connection to server 122.51.6.138/122.51.6.138:2181. Will not attempt to authenticate using SASL (unknown error)
2019-12-25 09:52:31.677  INFO 6882 --- [DubboZkclientConnector-SendThread(122.51.6.138:2181)] org.apache.zookeeper.ClientCnxn          : Socket connection established to 122.51.6.138/122.51.6.138:2181, initiating session
2019-12-25 09:52:31.716  WARN 6882 --- [DubboZkclientConnector-SendThread(122.51.6.138:2181)] org.apache.zookeeper.ClientCnxn          : Unable to reconnect to ZooKeeper service, session 0x10000004c5e0058 has expired
2019-12-25 09:52:31.716  INFO 6882 --- [DubboZkclientConnector-EventThread] org.I0Itec.zkclient.ZkClient             : zookeeper state changed (Expired)
2019-12-25 09:52:31.716  INFO 6882 --- [DubboZkclientConnector-SendThread(122.51.6.138:2181)] org.apache.zookeeper.ClientCnxn          : Unable to reconnect to ZooKeeper service, session 0x10000004c5e0058 has expired, closing socket connection
2019-12-25 09:52:31.717  INFO 6882 --- [DubboZkclientConnector-EventThread] org.apache.zookeeper.ZooKeeper           : Initiating client connection, connectString=122.51.6.138:2181 sessionTimeout=30000 watcher=org.I0Itec.zkclient.ZkClient@26807e7c
2019-12-25 09:52:31.729  INFO 6882 --- [DubboZkclientConnector-SendThread(122.51.6.138:2181)] org.apache.zookeeper.ClientCnxn          : Opening socket connection to server 122.51.6.138/122.51.6.138:2181. Will not attempt to authenticate using SASL (unknown error)
2019-12-25 09:52:31.730  INFO 6882 --- [DubboZkclientConnector-EventThread] org.apache.zookeeper.ClientCnxn          : EventThread shut down for session: 0x10000004c5e0058
2019-12-25 09:52:32.764  INFO 6882 --- [DubboZkclientConnector-SendThread(122.51.6.138:2181)] org.apache.zookeeper.ClientCnxn          : Socket connection established to 122.51.6.138/122.51.6.138:2181, initiating session
2019-12-25 09:52:32.981  INFO 6882 --- [DubboZkclientConnector-SendThread(122.51.6.138:2181)] org.apache.zookeeper.ClientCnxn          : Session establishment complete on server 122.51.6.138/122.51.6.138:2181, sessionid = 0x10000004c5e0059, negotiated timeout = 30000
2019-12-25 09:52:32.981  INFO 6882 --- [DubboZkclientConnector-EventThread] org.I0Itec.zkclient.ZkClient             : zookeeper state changed (SyncConnected)
2019-12-25 10:30:04.156  WARN 6882 --- [DubboZkclientConnector-SendThread(122.51.6.138:2181)] org.apache.zookeeper.ClientCnxn          : Client session timed out, have not heard from server in 63920ms for sessionid 0x10000004c5e0059
2019-12-25 10:30:04.940  INFO 6882 --- [DubboZkclientConnector-SendThread(122.51.6.138:2181)] org.apache.zookeeper.ClientCnxn          : Client session timed out, have not heard from server in 63920ms for sessionid 0x10000004c5e0059, closing socket connection and attempting reconnect
2019-12-25 10:30:05.044  INFO 6882 --- [DubboZkclientConnector-EventThread] org.I0Itec.zkclient.ZkClient             : zookeeper state changed (Disconnected)
2019-12-25 10:30:06.127  INFO 6882 --- [DubboZkclientConnector-SendThread(122.51.6.138:2181)] org.apache.zookeeper.ClientCnxn          : Opening socket connection to server 122.51.6.138/122.51.6.138:2181. Will not attempt to authenticate using SASL (unknown error)
2019-12-25 10:30:06.128 ERROR 6882 --- [DubboZkclientConnector-SendThread(122.51.6.138:2181)] o.apache.zookeeper.ClientCnxnSocketNIO   : Unable to open socket to 122.51.6.138/122.51.6.138:2181
2019-12-25 10:30:06.128  WARN 6882 --- [DubboZkclientConnector-SendThread(122.51.6.138:2181)] org.apache.zookeeper.ClientCnxn          : Session 0x10000004c5e0059 for server null, unexpected error, closing socket connection and attempting reconnect

java.net.SocketException: Network is unreachable
	at java.base/sun.nio.ch.Net.connect0(Native Method) ~[na:na]
	at java.base/sun.nio.ch.Net.connect(Net.java:482) ~[na:na]
	at java.base/sun.nio.ch.Net.connect(Net.java:474) ~[na:na]
	at java.base/sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:694) ~[na:na]
	at org.apache.zookeeper.ClientCnxnSocketNIO.registerAndConnect(ClientCnxnSocketNIO.java:277) ~[zookeeper-3.4.8.jar:3.4.8--1]
	at org.apache.zookeeper.ClientCnxnSocketNIO.connect(ClientCnxnSocketNIO.java:287) ~[zookeeper-3.4.8.jar:3.4.8--1]
	at org.apache.zookeeper.ClientCnxn$SendThread.startConnect(ClientCnxn.java:1024) ~[zookeeper-3.4.8.jar:3.4.8--1]
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1060) ~[zookeeper-3.4.8.jar:3.4.8--1]

2019-12-25 10:30:07.754  INFO 6882 --- [DubboZkclientConnector-SendThread(122.51.6.138:2181)] org.apache.zookeeper.ClientCnxn          : Opening socket connection to server 122.51.6.138/122.51.6.138:2181. Will not attempt to authenticate using SASL (unknown error)
2019-12-25 10:30:07.755 ERROR 6882 --- [DubboZkclientConnector-SendThread(122.51.6.138:2181)] o.apache.zookeeper.ClientCnxnSocketNIO   : Unable to open socket to 122.51.6.138/122.51.6.138:2181
2019-12-25 10:30:07.755  WARN 6882 --- [DubboZkclientConnector-SendThread(122.51.6.138:2181)] org.apache.zookeeper.ClientCnxn          : Session 0x10000004c5e0059 for server null, unexpected error, closing socket connection and attempting reconnect

java.net.SocketException: Network is unreachable
	at java.base/sun.nio.ch.Net.connect0(Native Method) ~[na:na]
	at java.base/sun.nio.ch.Net.connect(Net.java:482) ~[na:na]
	at java.base/sun.nio.ch.Net.connect(Net.java:474) ~[na:na]
	at java.base/sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:694) ~[na:na]
	at org.apache.zookeeper.ClientCnxnSocketNIO.registerAndConnect(ClientCnxnSocketNIO.java:277) ~[zookeeper-3.4.8.jar:3.4.8--1]
	at org.apache.zookeeper.ClientCnxnSocketNIO.connect(ClientCnxnSocketNIO.java:287) ~[zookeeper-3.4.8.jar:3.4.8--1]
	at org.apache.zookeeper.ClientCnxn$SendThread.startConnect(ClientCnxn.java:1024) ~[zookeeper-3.4.8.jar:3.4.8--1]
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1060) ~[zookeeper-3.4.8.jar:3.4.8--1]

2019-12-25 10:30:09.865  INFO 6882 --- [DubboZkclientConnector-SendThread(122.51.6.138:2181)] org.apache.zookeeper.ClientCnxn          : Opening socket connection to server 122.51.6.138/122.51.6.138:2181. Will not attempt to authenticate using SASL (unknown error)
2019-12-25 10:30:09.902  INFO 6882 --- [DubboZkclientConnector-SendThread(122.51.6.138:2181)] org.apache.zookeeper.ClientCnxn          : Socket connection established to 122.51.6.138/122.51.6.138:2181, initiating session
2019-12-25 10:30:09.940  WARN 6882 --- [DubboZkclientConnector-SendThread(122.51.6.138:2181)] org.apache.zookeeper.ClientCnxn          : Unable to reconnect to ZooKeeper service, session 0x10000004c5e0059 has expired
2019-12-25 10:30:09.940  INFO 6882 --- [DubboZkclientConnector-EventThread] org.I0Itec.zkclient.ZkClient             : zookeeper state changed (Expired)
2019-12-25 10:30:09.940  INFO 6882 --- [DubboZkclientConnector-SendThread(122.51.6.138:2181)] org.apache.zookeeper.ClientCnxn          : Unable to reconnect to ZooKeeper service, session 0x10000004c5e0059 has expired, closing socket connection
2019-12-25 10:30:09.940  INFO 6882 --- [DubboZkclientConnector-EventThread] org.apache.zookeeper.ZooKeeper           : Initiating client connection, connectString=122.51.6.138:2181 sessionTimeout=30000 watcher=org.I0Itec.zkclient.ZkClient@26807e7c
2019-12-25 10:30:09.943  INFO 6882 --- [DubboZkclientConnector-EventThread] org.apache.zookeeper.ClientCnxn          : EventThread shut down for session: 0x10000004c5e0059
2019-12-25 10:30:09.943  INFO 6882 --- [DubboZkclientConnector-SendThread(122.51.6.138:2181)] org.apache.zookeeper.ClientCnxn          : Opening socket connection to server 122.51.6.138/122.51.6.138:2181. Will not attempt to authenticate using SASL (unknown error)
2019-12-25 10:30:09.977  INFO 6882 --- [DubboZkclientConnector-SendThread(122.51.6.138:2181)] org.apache.zookeeper.ClientCnxn          : Socket connection established to 122.51.6.138/122.51.6.138:2181, initiating session
2019-12-25 10:30:10.015  INFO 6882 --- [DubboZkclientConnector-SendThread(122.51.6.138:2181)] org.apache.zookeeper.ClientCnxn          : Session establishment complete on server 122.51.6.138/122.51.6.138:2181, sessionid = 0x10000004c5e005a, negotiated timeout = 30000
2019-12-25 10:30:10.015  INFO 6882 --- [DubboZkclientConnector-EventThread] org.I0Itec.zkclient.ZkClient             : zookeeper state changed (SyncConnected)
2019-12-25 11:06:26.061  INFO 6882 --- [ZkClient-EventThread-61-122.51.6.138:2181] org.I0Itec.zkclient.ZkEventThread        : Terminate ZkClient event thread.
2019-12-25 11:06:26.068  INFO 6882 --- [Thread-7] ConfigServletWebServerApplicationContext : Closing org.springframework.boot.web.servlet.context.AnnotationConfigServletWebServerApplicationContext@65045a87: startup date [Sat Nov 16 15:56:45 CST 2019]; root of context hierarchy
2019-12-25 11:06:26.104  INFO 6882 --- [DubboZkclientConnector-EventThread] org.apache.zookeeper.ClientCnxn          : EventThread shut down for session: 0x10000004c5e005a
2019-12-25 11:06:26.104  INFO 6882 --- [DubboShutdownHook] org.apache.zookeeper.ZooKeeper           : Session: 0x10000004c5e005a closed
2019-12-25 11:06:26.128  INFO 6882 --- [Thread-7] o.s.j.e.a.AnnotationMBeanExporter        : Unregistering JMX-exposed beans on shutdown
2019-12-25 11:06:26.130  INFO 6882 --- [Thread-7] o.s.j.e.a.AnnotationMBeanExporter        : Unregistering JMX-exposed beans
2019-12-25 11:06:26.138  INFO 6882 --- [Thread-7] com.alibaba.druid.pool.DruidDataSource   : {dataSource-1} closed
